pkg/sentry/fsimpl/fuse/connection.go

```diff
@@ -73,11 +73,16 @@ func NewFUSEConnection(ctx context.Context, fd *vfs.FileDescription) *Connection
 	hdrLen := uint32((*linux.FUSEHeaderOut)(nil).SizeBytes())
 	fuseFD.writeBuf = make([]byte, hdrLen)
 	fuseFD.completions = make(map[linux.FUSEOpID]*FutureResponse)
+	fuseFD.requestKind = make(map[linux.FUSEOpID]linux.FUSEOpcode)
 	fuseFD.waitCh = make(chan struct{}, MaxInFlightRequests)
 	fuseFD.writeCursor = 0
 	fuseFD.readCursor = 0
 
-	return &Connection{fd: fuseFD}
+	conn := &Connection{
+		fd:fuseFD,
+	}
+	fs.fuseConn = conn
```

Comment:
changing a member field in a New constructor seems a little bit not intuitive enough. perhaps we can return both the *Connection and error?